### DISCUSSION TITLE
Why is this routing wrong? (Data-Issue or Software-Bug?)

### DISCUSSION THREAD

**Discussion by Nudin** [valhalla/valhalla#4765] [Q&A] [answered]
See this route, the issue appears for both bicycle and pedestrian routing. I see no reasons why "Holledauer Straße" should not be used. Is this a bug in valhalla or a bug in OSM data? I checked for unconnected points or wrong tags, but didn't found anything. Other routing engines don't show this oddity.
https://valhalla.openstreetmap.de/directions?profile=bicycle&wps=11.509879242661096%2C48.184966232284076%2C11.511975998777789%2C48.185091413410596
![Screenshot](https://github.com/valhalla/valhalla/assets/509205/2bfaefc6-b32f-4a46-ba9f-163fca9e3dfa)

**Comment by chrstnbwnkl** [comment, marked as answer]
Seems to be another case of #2563 (see also #4081 and #4320), [this way](http://www.openstreetmap.org/way/27828662) must be the culprit
